src/main/kotlin/net/lumalyte/lg/interaction/commands/GuildCommand.kt

```
@@ -1310,14 +1310,36 @@ class GuildCommand : BaseCommand(), KoinComponent {
-            player.sendMessage("§cGuild tag can only contain uppercase letters and numbers.")
+        if (tag.contains("<<") || tag.contains(">>")) {
+            player.sendMessage("§cInvalid tag syntax: double brackets.")
+            return
+        }
+
+        val miniMessage = net.kyori.adventure.text.minimessage.MiniMessage.miniMessage()
+        val visibleChars = try {
+            val component = miniMessage.deserialize(tag)
+            net.kyori.adventure.text.serializer.plain.PlainTextComponentSerializer.plainText()
+                .serialize(component).length
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)